nntrainer/layers/acti_func.h

```diff
@@ -667,6 +694,8 @@ class ActiFunc {
 private:
   constexpr static inline float alpha = 1.0f; /**< alpha for elu */
   constexpr static inline float beta = 1.0f;  /**< beta for Softplus */
+  constexpr static inline float selu_alpha = 1.67326324f; /**< alpha for selu */
+  constexpr static inline float selu_scale = 1.05070098f; /**< scale for selu */
```

Comment:
As you may already know, setting the parameters of the activation function in this way is not a good direction. I understand that existing codes, especially Leaky ReLU, have already been implemented in this manner. So I know why you implemented it this way and I understand you. As for the next task, rather than adding an activation function, how about improving the structure so that the parameters of the activation function can be processed by receiving the property as input? It is optional but not essential for you.

Reply:
Thanks for your opinion! I think that the task will be quite worthwhile.